Nodejs/Product/Nodejs/NodejsPackage.cs

```diff
@@ -661,6 +661,21 @@ internal sealed partial class NodejsPackage : CommonPackage {
             LogLooseFileAnalysisLevel();
         }
 
+        private VsProjectAnalyzer CreateLooseVsProjectAnalyzer() {
+            // In ES6 IntelliSense mode, rather than overriding the default JS editor,
+            // we throw to Salsa. If Salsa detects that it's not operating within the Node.js project context,
+            // it will throw to JSLS. This means that currently, loose files will be handled by the JSLS editor,
+            // rather than the NodeLS editor (which means that the loose project analyzer will not be invoked
+            // in these scenarios.)
+            //
+            // Because Salsa doesn't support the stitching together inputs from the surrounding text view,
+            // we do not throw to Salsa from the REPL window. However, in order to provide a workable editing
+            // experience within the REPL context, we initialize the loose analyzer with Quick IntelliSense
+            // during ES6 mode.
+            var analysisLevel = IntellisenseOptionsPage.AnalysisLevel == AnalysisLevel.Preview ? AnalysisLevel.NodeLsMedium : IntellisenseOptionsPage.AnalysisLevel;
+            return new VsProjectAnalyzer(analysisLevel, false);
```

Comment:
Just added #668 to our On Radar milestone so it doesn't fall through the cracks. We won't be addressing it for Dev14, but certainly for Dev15